code/modules/clothing/head/head.dm

```diff
@@ -362,7 +362,7 @@ GLOBAL_LIST_INIT(allowed_hat_items, list(
 	/obj/item/tool/pen = PREFIX_HAT_GARB_OVERRIDE,
 	/obj/item/tool/pen/blue = PREFIX_HAT_GARB_OVERRIDE,
 	/obj/item/tool/pen/red = PREFIX_HAT_GARB_OVERRIDE,
-	/obj/item/tool/pen/fountain = NO_GARB_OVERRIDE,
+	/obj/item/tool/pen/multicolor = NO_GARB_OVERRIDE,
```

Comment:
This is an incorrect override which is why the missing icons test is flagging this. `pen_black` for example does not exist in helmet_garb/misc.dmi but `hat_pen_black` and `helmet_pen_black` do. So to automatically prefix the state with `hat_` you need to use `PREFIX_HAT_GARB_OVERRIDE`
```suggestion
	/obj/item/tool/pen/multicolor = PREFIX_HAT_GARB_OVERRIDE,
```